connectors/src/resources/connector/confluence.ts

```diff
@@ -37,26 +38,30 @@ export class ConfluenceConnectorStrategy
     connector: ConnectorResource,
     transaction: Transaction
   ): Promise<void> {
-    await Promise.all([
-      ConfluenceConfiguration.destroy({
-        where: {
-          connectorId: connector.id,
-        },
-        transaction,
-      }),
-      ConfluenceSpace.destroy({
-        where: {
-          connectorId: connector.id,
-        },
-        transaction,
-      }),
-      ConfluencePage.destroy({
-        where: {
-          connectorId: connector.id,
-        },
-        transaction,
-      }),
-    ]);
+    await ConfluenceConfiguration.destroy({
+      where: {
+        connectorId: connector.id,
+      },
+      transaction,
+    });
+    await ConfluenceSpace.destroy({
+      where: {
+        connectorId: connector.id,
+      },
+      transaction,
+    });
+    await ConfluenceFolder.destroy({
+      where: {
+        connectorId: connector.id,
+      },
+      transaction,
+    });
+    await ConfluencePage.destroy({
+      where: {
+        connectorId: connector.id,
+      },
+      transaction,
+    });
```

Comment:
So I take it that doing it concurrently is problematic, since you moved away from that? Maybe add comment explaining why it shouldn't be done?